src/main/java/com/projectg/geyserupdater/bungee/util/GeyserBungeeDownloader.java

```diff
@@ -58,20 +58,12 @@ public static void updateGeyser() {
      * @return true if the download was successful, false if not.
      */
     private static boolean downloadGeyser() {
-        String fileUrl;
-        try {
-            fileUrl = "https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/" + GeyserProperties.getGeyserGitPropertiesValue("git.branch") + "/lastSuccessfulBuild/artifact/bootstrap/bungeecord/build/libs/Geyser-BungeeCord.jar";
-        } catch (IOException e) {
-            logger.error("Failed to get the current Geyser branch when attempting to download a new build of Geyser!");
-            e.printStackTrace();
-            return false;
-        }
+        String fileUrl = Constants.GEYSER_BASE_URL + Constants.GEYSER_DOWNLOAD_LINK + FileUtils.Platform.BUNGEECORD.getUrlComponent();
         String outputPath = "plugins/GeyserUpdater/BuildUpdate/Geyser-BungeeCord.jar";
         try {
-            FileUtils.downloadFile(fileUrl, outputPath);
+            FileUtils.downloadFile(fileUrl, outputPath, FileUtils.Platform.BUNGEECORD);
         } catch (IOException e) {
-            logger.error("Failed to download the newest build of Geyser");
-            e.printStackTrace();
+            logger.error("Failed to download the newest build of Geyser" + e.getMessage());
             return false;
         }
```

Comment:
1) Change to Exception
2) Can we still print the exception if the logger is in debug mode? If that exists in its current form.

applies to Spigot and Velocity downloader as well